EuclideanGeometry/Foundation/Axiom/Linear/Line.lean

```diff
@@ -194,7 +223,9 @@ theorem linear (l : Line P) {A B C : P} (h₁ : A LiesOn l) (h₂ : B LiesOn l)
       | inl c => sorry
       | inr c => sorry
 
-theorem maximal (l : Line P) {A B : P} (h₁ : A ∈ l.carrier) (h₂ : B ∈ l.carrier) (h : B ≠ A) : (∀ (C : P), colinear A B C → (C ∈ l.carrier)) := sorry
+theorem maximal' {l : Line P} {A B : P} (h₁ : A LiesOn l) (h₂ : B LiesOn l) (h : B ≠ A) : (∀ (C : P), colinear A B C → (C LiesOn l)) := by 
+  intro C Co
+  sorry
```

Comment:
this theorem can be reforulate even better. we can move C，colinear A B C into hypothesis.

Reply:
This theorem was written by otheres. So our team didn't make changes.